soap/src/java/com/zimbra/soap/admin/message/ResetAccountPasswordRequest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra Collaboration Suite Server
+ * Copyright (C) 2020 Synacor, Inc.
```

Comment:
change the year to 2021 here and other files

Reply:
done